src/Kathara/setting/addon/KubernetesSettingsAddon.py

```diff
@@ -6,23 +6,26 @@
     "api_server_url": "Empty String",
     "api_token": "Empty String",
     "host_shared": True,
-    "image_pull_policy": "IfNotPresent"
+    "image_pull_policy": "IfNotPresent",
+    "private_registry_dockerconfigjson": "Empty String"

```

Comment:
Same here, use a better name for example `docker_private_registry_json`

Reply:
See above